.github/workflows/verify-non-code.yaml

```diff
@@ -0,0 +1,51 @@
+---
+name: non-coding-checks
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+on:
+  push:
+    branches: ['*']
```

Comment:
we dont need to run this workflow on the push , can we remove it?

Reply:
done.